### PR TITLE
I16 gcs verify hash

### LINKED ISSUE #16: gcs verify hash
Verify hash when uploading or downloading objects from gcs.

### PR DESCRIPTION
**Pull request by martinkostal**
Upload and download hash veerification

### CODE REVIEW

File: cloudwrapper/gcs.py [hunk 1]
```diff
@@ -4,7 +4,11 @@
 Author: Martin Mikita <martin.mikita@klokantech.com>
 """
 
+import base64
+import crc32c
```
Comment (MartinMikita) [inline cloudwrapper/gcs.py]: move this import lower, between `try + raise ImportError` and add name of the package for installing via PIP

**Review by MartinMikita — CHANGES_REQUESTED** [review]
Perfect, small change to improve portability and we can merge into Cloudwrapper!

File: cloudwrapper/gcs.py [hunk 2]
```diff
@@ -20,6 +19,7 @@
     from warnings import warn
```
Comment (MartinMikita) [inline cloudwrapper/gcs.py:20]: missing `import crc32c` before `except ImportError`

File: cloudwrapper/gcs.py [hunk 3]
```diff
@@ -20,6 +19,7 @@
...
     install_modules = [
         'gcloud==0.18.3',
+	'crc32c==2.0.1',
```
Comment (MartinMikita) [inline cloudwrapper/gcs.py]: please, use spaces, not tabs

**Review by MartinMikita — CHANGES_REQUESTED** [review]

File: cloudwrapper/gcs.py (hunk 2 above)
Comment (martinkostal) [reply cloudwrapper/gcs.py:20]: Done.

Comment (martinkostal) [reply cloudwrapper/gcs.py:20]: Done.

File: cloudwrapper/gcs.py (hunk 3 above)
Comment (martinkostal) [reply cloudwrapper/gcs.py]: Done.

File: cloudwrapper/gcs.py (hunk 1 above)
Comment (martinkostal) [reply cloudwrapper/gcs.py]: Done.

**Review by MartinMikita — APPROVED** [review]

**Pull request merged**